lib/neurx/structures/connection.ex

```diff
@@ -9,7 +9,7 @@ defmodule Neurx.Connection do
 
   alias Neurx.{Connection}
 
-  defstruct pid: nil, source_pid: nil, target_pid: nil, weight: 0.4
+  defstruct pid: nil, source_pid: nil, target_pid: nil, weight: (1 - :random.uniform())
```

Comment:
:random is depricated.

Please use :rand [http://erlang.org/doc/man/rand.html](http://erlang.org/doc/man/rand.html)

it will take 1 line to seed, then 1 line to generate.

Reply:
More lines?  :(